types/signed_header.go

```diff
@@ -0,0 +1,52 @@
+package types
+
+import (
+	"encoding"
+	"time"
+
+	"github.com/celestiaorg/go-header"
+)
+
+func (sH *SignedHeader) New() header.Header {
+	return new(SignedHeader)
+}
+
+func (sH *SignedHeader) Hash() Hash {
+	return sH.Header.Hash()
```

Comment:
This method is not strictly required if `Header` is being embedded, because caller can use `SignedHeader.Hash()` directly and it will pick up `Header.Hash`.

Reply:
thanks for your suggestion. removed the unnecessary methods.